test/code_fund/themes/themes_test.ex

```diff
@@ -0,0 +1,43 @@
+defmodule CodeFund.ThemesTest do
+  use CodeFund.DataCase
+  alias CodeFund.Themes
+
+  setup do
+    user = CodeFund.Support.Fixture.generate(:user)
```

Comment:
I've been swapping my fixtures out w/ ExMachina, can we do that here?

Reply:
Definintely